app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+def cache(func: None) -> None:
```

Comment:
You should change typing for `cache`, because `func` will be `Callable`, as well as the returning `wrapper` will be `Callable`.